---
layout: page
permalink: /projects/
title: "Projects"
modified: 2014-08-07 16:25
tags: []
image:
  feature: 
  credit: 
  creditlink: 
share: 
---

Linked Data crawler for Metagenomics data (Development)
==================

Linked Data (http://www.w3.org/standards/semanticweb/data) is widely used to publish Data on the Web in a machine friendly manner, resulting in the Linked Open Data (LOD) cloud (http://lod-cloud.net/). Life Sciences data comprisses a considerable part of the LOD cloud, including (possibly) Metagenomics data. The aim of this project is to develop Web crawler that automatically browses Linked Data datasets of the LOD and retrieves Metagenomics related data. The crawler should be configurable and Metagenomics should be the test case: the crawler will be the outcome of the project, regardless of the Metagenomics data retrieved (it is acceptable for the crawler to retrieve non-significant data, quality or quantity wise, if it is demonstrated that it is not a fault of the crawling process).

Tutor: Mikel Egaña Aranguren (mikel.egana@ehu.es)

Methylation data analysis application (Development)
==================

R is one of the most widely used computational environments used to analyze biological data. In the particular case of methylation data, there are several packages that provide a wide variety of tools to preprocess, visualize and analyze different types of data. Although the number of biologists that are able to use R is increasing, a simple, visual interface for these packages can be of great help. In that regard, it is worth of noting the shiny package that, with only a bit of  programing can be used to create nice looking web applications that run R code. The goal of this project is to develop, using shiny, a web application that can be used to preprocess, visualize and analyze methylation data.

Tutor: Borja Calvo (borja.calvo@ehu.es)

Comparison of NGS DNA methylation profiles of different types of pluripotent cells (Research)
==================

In the field of cellular reprograming, a lot of research has been performed in the last years in the search for the so called “Epigenetic somatic memory” and several controversies have arisen in relation to its existence and biological impact. Several researchers have tackled the problem searching for Differentially Methylated Regions (DMRs) between ESCs and iPSCs.

In this project several DNA methylation profiles form Next Generation Sequencing (NGS) from fibroblasts, Embryonic Stems Cells (ESCs) and Induced Reprogrammed Stems Cells (iPSCs) will be download from public databases and it will be analyzed the DNA methylation differences between them.

For such task it will be used the program P3BSseq (Parallel processing pipeline for analysis of bisulfite sequencing data) that can be download from https://sourceforge.net/projects/p3bsseq/

Tutor: Marcos Arauzo (mararabra@yahoo.co.uk)



